frontend/src/index.html

```diff
@@ -2,29 +2,24 @@
 <html lang="en">
 
 <head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>NixOS Search</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>NixOS Search</title>
 
-  <script type="text/javascript" src="https://nixos.org/js/jquery.min.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```

Comment:
Can we remove the duplicate `<meta name=viewport`?

Reply:
Yes, missed that one.